src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -334,6 +334,13 @@ public static boolean isGenomicsDBPath(final String path) {
                 throw new GATKException("Unable to automatically instantiate codec " + codecClass.getName());
             }
         }
+
+        // Set the name if we have to:
+        if ( codec instanceof NameAwareCodec ) {
+            final NameAwareCodec namedCodec = (NameAwareCodec) codec;
+            namedCodec.setName( featureInput.getFeaturePath() );
```

Comment:
Can you set the codec's name to `featureInput.getName()` instead, so that if a tag is present it will be used as the name?

Reply:
Sure.  Fixed.